XamlControlsGallery/ControlPages/TabViewPage.xaml

```diff
@@ -148,10 +148,10 @@
 
                     <muxc:TabView SelectedIndex="0" Margin="-12" AddTabButtonClick="TabView_AddButtonClick" TabCloseRequested="TabView_TabCloseRequested" Loaded="TabView_Loaded" MinHeight="475" TabWidthMode="SizeToContent">
                         <muxc:TabView.TabStripHeader>
-                            <TextBlock Text="TabStripHeader Content" VerticalAlignment="Center" Margin="8,6" Style="{ThemeResource BaseTextBlockStyle}" />
+                            <TextBlock Text="TabStripHeader Content" VerticalAlignment="Center" Margin="8,8" Style="{ThemeResource BaseTextBlockStyle}" />
                         </muxc:TabView.TabStripHeader>
                         <muxc:TabView.TabStripFooter>
-                            <TextBlock Text="TabStripFooter Content" VerticalAlignment="Center" HorizontalAlignment="Right" Margin="6" Style="{ThemeResource BaseTextBlockStyle}" />
+                            <TextBlock Text="TabStripFooter Content" VerticalAlignment="Center" HorizontalAlignment="Right" Margin="8" Style="{ThemeResource BaseTextBlockStyle}" />
```

Comment:
Similarly, I worry about messing with the alignment inside the tab strip too much. @stmoy, any toughts?